src/pages/[platform]/ai/conversation/connect-your-frontend/index.mdx

```diff
@@ -0,0 +1,451 @@
+| `metadata`       | `Record<string, string> \| undefined` | Metadata associated with the conversation.<br/>You can specify arbitrary metadata when creating or updating a conversation. |
+| `createdAt`      | `string` | The date and time when the conversation was created |w
+| `updatedAt`      | `string` | The date and time when the last user message was sent |
+| `sendMessage()`  | `(content: MessageContent) => {`<br/>&nbsp;&nbsp;`data: Message;`<br/>&nbsp;&nbsp;`errors: Error[]`<br/>`}` | Send a message to the AI assistant |
+| `listMessages()` | `() => {`<br/>&nbsp;&nbsp;`data: Message[];`<br/>&nbsp;&nbsp;`errors: Error[]`<br/>`}` | List all messages for the conversation |
+| `onStreamEvent()`| `(options: {`<br/>&nbsp;&nbsp;`next: (event: StreamEvent) => void;`<br/>&nbsp;&nbsp;`error: (error: Error) => void;`<br/>`}) => void` | Subscribe to assistant responses |
+
+### Message
+
+A `Message` is a single chat message between an application user and an LLM. Each message has a `role` property that indicates whether the message is from the user or the assistant. User and assistant messages have a one-to-one relationship. Assistant messages contain an `associatedUserMessageId` property that points to the `id` of the user message that triggered the assistant response.
+
+The `Message` type is accessible via `Schema['myChat']['messageType']`.
+
+| Property  | Type   | Description  |
+|-----------|:-------|:-------------|
+| `id` | `string` | The unique identifier for the message |
+| `conversationId` | `string` | The ID of the conversation this message belongs to |
+| `associatedUserMessageId` | `string \| undefined` | For assistant messages, the ID of the user message that triggered the response |
+| `content` | `MessageContent[]` | The content of the message |
+| `role` | `'user' \| 'assistant'` | Whether the message is from the user or assistant |
+| `createdAt` | `string` | The date and time when the message was created |
+
+## Data flow
+
+1. Create a new conversation with `.create()` or get an existing one with `.get()`.
+2. Subscribe to assistant responses for a conversation with `.onStreamEvent()`.
+3. Send messages to the conversation with `.sendMessage()`.
+
+```ts
+// 1. Create a conversation
+const { data: chat, errors } = await client.conversations.chat.create();
+
+// 2. Subscribe to assistant responses
+const subscription = chat.onStreamEvent({
+  next: (event) => {
+    console.log(event);
+  },
+  error: (error) => {
+    console.error(error);
+  },
+});
+
+// 3. Send a message to the conversation
+const { data: message, errors } = await chat.sendMessage('Hello, world!');
+```
+
+## Managing conversations
+
+### Create a conversation
+
+Conversation routes defined in your schema are accessible via the `.conversations` namespace. First create a client instance with the `generateClient()` function.
```

Comment:
Maybe this should go above the previous section since its the first time in the page using the client?

Reply:
Addressed in [53184f1](https://github.com/aws-amplify/docs/pull/8119/commits/53184f1afeb0c582cb6bf3147d83c1b131f11e01)